Source/WebKit/Shared/ios/WebIOSEventFactory.mm

```diff
@@ -145,57 +147,71 @@
     int clickCount = 0;
     double timestamp = event.timestamp;
 
-    return WebKit::WebMouseEvent({ type, OptionSet<WebKit::WebEventModifier> { }, WallTime::fromRawSeconds(timestamp) }, button, buttons, position, position, deltaX, deltaY, deltaZ, clickCount);
+    return WebMouseEvent({ type, OptionSet<WebEventModifier> { }, WallTime::fromRawSeconds(timestamp) }, button, buttons, position, position, deltaX, deltaY, deltaZ, clickCount);
 }
 
 #if HAVE(UISCROLLVIEW_ASYNCHRONOUS_SCROLL_EVENT_HANDLING)
-static WebKit::WebWheelEvent::Phase toWebPhase(UIScrollPhase phase)
+static WebWheelEvent::Phase toWebPhase(WKSEScrollViewScrollUpdatePhase phase)
 {
     switch (phase) {
+#if !SERVICE_EXTENSIONS_SCROLL_VIEW_IS_AVAILABLE
     case UIScrollPhaseNone:
```

Comment:
Matching `WKSEScrollViewScrollUpdatePhase` values against `UIScrollPhase` literals?

Reply:
Okay, I see what happened here. I thought the compiler would complain but they're all just integral values. 🤷

Reply:
Indeed, `UIScrollPhaseNone` does not have a `WKSE`-prefixed equivalent. In the case where `SERVICE_EXTENSIONS_SCROLL_VIEW_IS_AVAILABLE` is off, `WKSEScrollViewScrollUpdatePhase` is typedef'd to be `UIScrollPhase`, so this is identical to shipping code.